scripts/install-zenml-dev.sh

```diff
@@ -28,8 +28,10 @@ parse_args () {
 
 install_zenml() {
     # install ZenML in editable mode
+    uv pip install $PIP_ARGS -e ".[server,templates,terraform,secrets-aws,secrets-gcp,secrets-azure,secrets-hashicorp,s3fs,gcsfs,adlfs,dev]"
 
-    uv pip install $PIP_ARGS -e ".[server,templates,terraform,secrets-aws,secrets-gcp,secrets-azure,secrets-hashicorp,s3fs,gcsfs,adlfs,dev,mlstacks]"
+    # TODO: Remove. Temporary installation from the upgrade branch.
```

Comment:
TODO

Reply:
I am aware of this of these TODOs. I added them to the list in the description. For the sake of clarity, resolving these two.